apps/lambda_function_api_gateway_all_methods_passthrough/api_gateway.tf

```diff
@@ -69,28 +69,63 @@ resource "aws_lambda_permission" "allow-api-gateway-parent-resource-get" {
   source_arn = "arn:aws:execute-api:${var.region}:${module.aws_core_data.account_id}:${aws_api_gateway_rest_api.app.id}/*/${aws_api_gateway_method.app.http_method}${aws_api_gateway_resource.app.path}"
 }
 
-resource "aws_api_gateway_api_key" "app" {
-  name = "${var.app_name}"
+# swagger
+resource "aws_api_gateway_resource" "swagger_parent" {
```

Comment:
Would it be possible to create a module out of this? I see some copy and paste behavior :)